subsys/bluetooth/controller/ticker/ticker.c

```diff
@@ -205,27 +205,12 @@ BUILD_ASSERT(sizeof(struct ticker_user_op) == TICKER_USER_OP_T_SIZE);
 #define TICKER_INSTANCE_MAX 1
 static struct ticker_instance _instance[TICKER_INSTANCE_MAX];
 
+#define INC_MOD(y, x) ((y) + 1 == (x) ? 0 : (y) + 1)
```

Comment:
Is there a Zephyr wide macro available for this that can be used? To be consistent, a zephyr wide use will be recommended.

Reply:
There is MODULO_INC() defined in 3 other modules but not exported. :D